### DISCUSSION TITLE
VSCode Extension

### DISCUSSION THREAD

**Discussion by dodeyeikoi** [Cyfrin/foundry-full-course-cu#1058] [General]
Can anyone please suggest any auto completion vscode extension that supports solidity? 

I'm not able to subscribe for github copilot at the moment and codeium is refusing to load.
@PatrickAlphaC

**Comment by theirrationalone** [comment]
Sorry to say.... there's no efficient substitutes than `github copilot`.... You can subscribe it for free by registering it as a student. [Go here](https://education.github.com/pack)

**Reply by dodeyeikoi** [reply]
Alright thanks for the help
